v4/export/dump.go

```diff
@@ -250,22 +255,34 @@ func dumpDatabases(ctx context.Context, conf *Config, connectPool *connectionsPo
 			}
 			for _, tableIR := range tableDataIRArray {
 				tableIR := tableIR
-				g.Go(func() error {
-					conn := connectPool.getConn()
-					defer connectPool.releaseConn(conn)
-					err := tableIR.Start(ctx, conn)
+				g.Add(1)
+				go func() {
+					defer g.Done()
+					retryTime := 1
```

Comment:
wait, why can't we keep using errgroup here?

Reply:
You are right. I have rewritten this part's logic using `errgroup` again.